common/version/src/lib.rs

```diff
@@ -0,0 +1,60 @@
+use std::fmt::{Display, Formatter};
+use std::str::FromStr;
+
+pub const DEFAULT_COMMIT_ID: &str = "unknown";
+
+#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
+pub struct Version {
+    inner: semver::Version,
+}
```

Comment:
Why not use `semver::Version` directly? This wrapper doesn't provide any extra functionality, and has some unfamiliar APIs with undocumented panics.

Reply:
This can be considered a nit.